### DISCUSSION TITLE
Translate "Visitor" on chat opened

### DISCUSSION THREAD

**Discussion by Broninn** [LiveHelperChat/livehelperchat#2029] [Q&A]
Hello,

I'm usying bot, and I need to translate the word "Visitor", but I couldn't find where I can do that.

![image](https://github.com/LiveHelperChat/livehelperchat/assets/65509910/1cab17a8-3c40-4f27-bd16-43209d516bdb)

It's likely a minor configuration issue, but I've searched for 2 days and haven't found a solution.

**Comment by remdex** [comment]
This is not translatable, but using Widget themes you can hide this completely.
